neurom/core/_neuron.py

```diff
@@ -70,6 +80,8 @@ def iter_neurites(obj, mapfun=None, filt=None):
     '''
     neurites = ((obj,) if isinstance(obj, Neurite) else
                 obj.neurites if hasattr(obj, 'neurites') else obj)
+    if iterator_type == NeuriteIter.NRN:
+        neurites = sorted(neurites, key=lambda neurite: NRN_ORDER.get(neurite.type, 5))
```

Comment:
the 5 creates a coupling to the NRN_ORDER that's not obvious, maybe do something like `max_value = max(NRN_ORDER.values()) + 1` previously, and use `....get(neurite.type, max_value)`

Reply:
Done